packages/react/src/primitives/Tabs/TabsItem.tsx

```diff
@@ -12,28 +12,32 @@ import { View } from '../View';
 import { primitiveWithForwardRef } from '../utils/primitiveWithForwardRef';
 import { BaseTabsItemProps, TabsItemProps } from './types';
 import { TabsContext } from './TabsContext';
+import { WHITESPACE_VALUE } from './constants';
 
 const TabsItemPrimitive: Primitive<TabsItemProps, 'button'> = (
   { className, value, children, onClick, as = 'button', role = 'tab', ...rest },
   ref
 ) => {
-  const { activeTab, setActiveTab } = React.useContext(TabsContext);
+  const { activeTab, setActiveTab, groupId } = React.useContext(TabsContext);
+  let idValue = value;
+  if (typeof idValue === 'string') {
+    idValue = idValue.replace(' ', WHITESPACE_VALUE);
+  }
   const isActive = activeTab === value;
   const handleOnClick = (e: React.MouseEvent<HTMLButtonElement>) => {
     if (isTypedFunction(onClick)) {
       onClick?.(e);
     }
     setActiveTab(value);
   };
-
   return (
     <View
       {...rest}
       role={role}
       as={as}
-      id={`${value}-tab`}
+      id={`${groupId}-tab-${idValue}`}
       aria-selected={isActive}
-      aria-controls={`${value}-panel`}
+      aria-controls={`${groupId}-panel-${idValue}`}
```

Comment:
`value` for `TabsPanel` and `TabsItem` is required to be a string

Reply:
From the testing I've done this change does not seem to be breaking. At this stage of the Tabs composition `value` is required to be defined, so creating `idValue` based on it does not seem to introduce any errors. Because `value` itself is not changed, as long as `defaultValue` matches the intended target Tab's `value` the controllability and default functionality seems to work as intended. Tested by changing `value` and `defaultValue` for the examples to include whitespace.